cli/lib/api.js

```diff
@@ -0,0 +1,21 @@
+const fetch = require('node-fetch');
+
+class Api {
+  constructor(baseUrl) {
+    this.baseUrl = baseUrl;
+  }
+
+  headers(token) {
```

Comment:
maybe then it makes sense to also load `.entropicrc` in API constructor and for module to export API instance you can copy\update rather than a class?